### PR TITLE
30/people section

### PR DESCRIPTION
**Pull request by George-Onyango**
# Description

This is a pull request that fixes issue 30 creating the counter of investor and careers started

Fixes # (issue) - (#30 )

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# Screenshot (optional)
Screenshot from 2023-02-17 09-59-17

Mobile
Screenshot from 2023-02-17 09-59-10

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 
Please also list any relevant details for your test configuration if necessary.
- [x] Test against responsive screen 

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by George-Onyango** [comment]
@IanOdhiambo9 requesting review

File: libs/pages/elewa/social-impact/src/lib/pages-elewa-social-impact.module.ts [hunk 1]
```diff
@@ -1,13 +1,17 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
-import { LayoutModule } from '@elewa-group/elements/layout';
+
+import { PeopleSectionComponent } from './component/people-section/people-section.component';
+
 import { SocialImpactPageComponent } from './pages/social-impact-page/social-impact-page.component';
-import { SocialImpactHeroSectionComponent } from './components/social-impact-hero-section/social-impact-hero-section.component';
+
 import { SocialImpactRoutingModule } from './social-impact.routing';
```
Comment (LemmyMwaura) [inline libs/pages/elewa/social-impact/src/lib/pages-elewa-social-impact.module.ts:8]: Routing Module comes last

File: libs/pages/elewa/social-impact/tsconfig.lib.json [hunk 2]
```diff
@@ -30,5 +30,8 @@
     "jest.config.ts",
     "src/**/*.test.ts"
   ],
-  "include": ["src/**/*.ts"]
+
+  // "include": ["src/**/*.ts"]
```
Comment (LemmyMwaura) [inline libs/pages/elewa/social-impact/tsconfig.lib.json:34]: remove this

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Design does not match the expectation. Please put some effort on your design.

File: libs/pages/elewa/social-impact/src/lib/pages-elewa-social-impact.module.ts [hunk 3]
```diff
@@ -1,13 +1,17 @@
...
+// eslint-disable-next-line @nrwl/nx/enforce-module-boundaries
+import { ElewaGroupImageAndTextBannerComponent } from 'libs/features/components/banners/src/lib/banners/elewa-group-image-and-text-banner/elewa-group-image-and-text-banner.component';
```
Comment (LemmyMwaura) [inline libs/pages/elewa/social-impact/src/lib/pages-elewa-social-impact.module.ts:10]: import from the aliased path.

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]
I can't see the image and text banner in the screenshots

**Comment by IanOdhiambo9** [comment]
I'm closing PR because this issue has been closed

**Pull request closed without merging**